stdlib/2and3/ssl.pyi

```diff
@@ -52,9 +53,9 @@ if sys.version_info < (3,) or sys.version_info >= (3, 4):
     def create_default_context(purpose: Any = ..., *,
                                cafile: Optional[str] = ...,
                                capath: Optional[str] = ...,
-                               cadata: Union[str, bytes, None] = ...) -> SSLContext: ...
+                               cadata: Union[Text, bytes, None] = ...) -> SSLContext: ...
 
-if sys.version_info >= (3, 4):
+if sys.version_info < (3,) or sys.version_info >= (3, 4):
```

Comment:
Please just remove this version check. We only support Python 2.7 and 3.5+, so it is not necessary anymore.

Reply:
Done.